### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by Itswali**
In this PR:
* I create a collection that keep a list of book
* I create a js function to store date in local storage
* I add a function to add book detail in html file as user  enter 
* I add a function to remove the book as user remove.
* I store the detail in localstorage
* I add linters to check for errors

### CODE REVIEW

**Review by levy002 — CHANGES_REQUESTED** [review]
Hello @Itswali 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have a descriptive README, and know that while this is a pair programming project your partner should contribute to this project as well.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md [hunk 1]
```diff
@@ -1 +1,65 @@
...
+
+- GitHub: @Itswali
+- Twitter: @Wali-Muhammad
+- LinkedIn: @wali-Muhammad
+
+👤 **Ivan Martinez von Halle**
+
+- GitHub: [@ivanmvh](https://github.com/ivanmvh)
+- Twitter: [@imprivado](https://twitter.com/imprivado)
+- LinkedIn: [Iván Martínez von Halle] (https://www.linkedin.com/in/ivan-martinez-von-halle/)
```
Comment (ElsonOtake) [inline README.md]: - [OPTIONAL] I think it's better to remove the space between `]` and `(`. The styling of your link in [markdown language](https://www.markdownguide.org/cheat-sheet/) is not working.

File: README.md [hunk 2]
```diff
@@ -1 +1,65 @@
...
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+<!-- ## Acknowledgments
```
Comment (ElsonOtake) [inline README.md]: - [OPTIONAL] I think removing unused comments from your files is better. I suggest creating the `Acknowledgements` section quoting those you want to thank. Your README will look more professional to those who check your work.

File: main.js [hunk 3]
```diff
@@ -0,0 +1,78 @@
+// constant variable declare
```
Comment (ElsonOtake) [inline main.js]: - [ ] I think it's better to create a collection that keeps a list of books (hint: you can use an array of objects for that). It's a project requirement. You are using the local storage to do this, but the local storage must be used to save your data for the next time you use the app.

File: main.js [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+  adder.innerHTML += `
+    <div> 
+    <p>${title.value}</p>
+    <p>${author.value}</P>
+    <button class ="remove" name="${title.value}">Remove book </button> 
+    <hr>
+    </div>
+    `;
+
+  // adding book with local storage
```
Comment (ElsonOtake) [inline main.js]: - [ ] Please create a code to add data to the collection that keeps a list of books. It's a project requirement.

File: main.js [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+addButton.addEventListener('click', (e) => {
+  e.preventDefault();
+
+  // array usage
+
+  adder.innerHTML += `
+    <div> 
+    <p>${title.value}</p>
+    <p>${author.value}</P>
+    <button class ="remove" name="${title.value}">Remove book </button>
```
Comment (ElsonOtake) [inline main.js]: - [ ] Please fix the functionality of this button. It's not working.

File: main.js [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+//  Remove book from shelf using array filter
+
+const remove = document.querySelectorAll('.remove');
+
+remove.forEach((item) => {
+  item.addEventListener('click', () => {
+    item.parentElement.remove();
+    const bookname = item.name;
+
+    // Remove book-info from local storage
```
Comment (ElsonOtake) [inline main.js]: - [ ] Please create the code to remove the book from the collection. It's a project requirement.

File: main.js [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+
+const addButton = document.getElementById('Add');
+const title = document.getElementById('title');
+const author = document.getElementById('author');
+const adder = document.getElementById('keeper');
+
+// setup the local storage
+
+if (localStorage.getItem('books') !== null) {
+  const getbook = JSON.parse(localStorage.getItem('books'));
```
Comment (ElsonOtake) [inline main.js]: - [ ] I think it's better to create a code to present the data from the collection. Load the collection with the data from the local storage when starting. Your code will look more professional and with improved functionality.

**Review by ElsonOtake — CHANGES_REQUESTED** [review]
STATUS: CHANGES REQUIRED 🟡 

Hi @Itswali @ivanmvh,

Good job so far! 🎉 🎉 
There are some issues that you still need to work on to go to the next project but you are almost there! 🥲 🥲 

woohoo-youre-almost

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️
- PR has a good title and summary. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by christianbiring1 — APPROVED** [review]
Hi @Itswali , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
# STATUS: APPROVED :tada: 
congraaaa

## Highlights
- [ ] All linters are green :heavy_check_mark: 
- [ ] Descriptive PR `title` and `summary`
- [ ] Good commit message
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**